tests/ui/delegation/glob-glob.rs

```diff
@@ -31,6 +33,6 @@ impl Trait for u8 {
 
 fn main() {
     let u = 0u8;
-    u.foo();
-    u.bar();
+    <u8 as Trait>::foo(&u);
+    <u8 as Trait>::bar(&u);
```

Comment:
What is this change for?

Reply:
`TraitFoo` and `TraitBar` are not in scope for method call in `Trait` impl. So I import these libraries, which leads to ambiguity.

Reply:
`reuse inner::TraitFoo::*;` should not require `TraitFoo` being in scope.
If this is a side effect of bodies being converted to method calls, then this side effect should be eliminated somehow, e.g. by adding the necessary traits to `trait_map` automatically or something like that.